public/app/plugins/panel/state-timeline/StateTimelinePanel.tsx

```diff
@@ -60,90 +68,117 @@ export const StateTimelinePanel = ({
     );
   }
 
-  if (frames.length === 1) {
-    const packet = getLastStreamingDataFramePacket(frames[0]);
-    if (packet) {
-      // console.log('STREAM Packet', packet);
-    }
```

Comment:
I'm not sure the `if (frames.length === `)` bit is still relevant, considering I've changed how `prepareTimelineFields` work. I looked at the commit history and it [originated here](https://github.com/grafana/grafana/pull/35465#discussion_r655945569). It looks stale to me (3y ago it was going to be used, but still not used), so I decided to just remove it.